resources/test/fixtures/pyupgrade/UP002.py

```diff
@@ -0,0 +1,4 @@
+from typing import Text
+
+def print_word(word: Text) -> None:
+    print(word)
```

Comment:
Nit: add a test case for `def print_word(word: typing.Text) -> None: print(word)`?

Reply:
Good catch, this does not work.

Reply:
Fixed!